doc/src/manual/documentation.md

```diff
@@ -81,7 +81,23 @@ As in the example above, we recommend following some simple conventions when wri
    ...
    """
    ```
-5. Include any code examples in an `# Examples` section.
+5. Provide hints to similar functions
+   
+   Sometimes there are functions of similar functionality, which are named quite different. To increase 
+   discoverability please provide a short list of these similar functions in a `See also: ` section.
+   
+   ```
+   See also: bar!, baz, baaz
+   ```
+   If the difference in functionality is not obvious from the name, you can provide a short description 
```

Comment:
I don't like this second part. Docstrings should be distinct and describe the function, not describe other functions. If similar functions are listed it is easy enough to look up what they do.

Reply:
I did not like that part either. I proposed this list as it seems to be common practice to refer other functions with a short description in docstrings:
```
 Sort a multidimensional array A along the given dimension. See sort! for a description of possible keyword arguments.
```

```
  shuffle([rng=GLOBAL_RNG,] v)

  Return a randomly permuted copy of v. The optional rng argument specifies a random number generator (see Random Numbers). To
  permute v in-place, see shuffle!. To obtain randomly permuted indices, see randperm.
```

Perhaps the better approach is to remove these redundant descriptions and just rely on the `See also:`-list

Reply:
The difference is that the examples you listed here refer to more documentation about the documented function -- they don’t describe other functions.

Reply:
At least the docstring to shuffle does: 
```
 To permute v in-place, see shuffle!. To obtain randomly permuted indices, see randperm.
```
And i think i have seen this pattern quite often.

Should we change those docs?

Reply:
I think that can be decided from case to case, and those examples are probably there for a reason. I just don't think we should explicitly recommend that style, as suggested in this PR.